### PR TITLE
feat:Add empty text when flags are empty

### LINKED ISSUE #3: flag list screen
go to https://app.launchdarkly.com/ and sign up.

Screen Shot 2020-10-29 at 12 04 26 PM

### PR DESCRIPTION
**Pull request by killbond007**
Capture d’écran 2021-03-17 à 23 18 09

Fixes #3

### CODE REVIEW

File: src/shared/feature-flags/flags-status-table.tsx [hunk 1]
```diff
@@ -55,15 +59,29 @@ export const FlagsStatusTable: React.FC<Props> = ({ data }) => {
...
+      <NewFlagController newFlaglabel="+ Flag" />
       <div>
         Use this page to see all feature flags in this project. Select a flag to
         manage the environment-specific targeting and rollout rules.
       </div>
-      <Table<FlagsStatus_flagsStatus_flags>
-        columns={columns}
-        dataSource={data}
-      />
+      <ConfigProvider
```
Comment (bitQ2019) [inline src/shared/feature-flags/flags-status-table.tsx:67]: why use ConfigProvider here?

File: src/shared/feature-flags/new-flag-controller.tsx [hunk 2]
```diff
@@ -3,7 +3,11 @@ import React from "react";
 import { NewFlagModal } from "./new-flag-modal";
 import { useUpsertFlag } from "../flag-details/hooks/use-upsert-flag";
 
-export const NewFlagController: React.FC = () => {
+type Props = {
+  newFlaglabel: string;
```
Comment (bitQ2019) [inline src/shared/feature-flags/new-flag-controller.tsx:7]: newFlagLabel

**Review by bitQ2019 — CHANGES_REQUESTED** [review]
plz see the comments

File: src/shared/feature-flags/flags-status-table.tsx (hunk 1 above)
Comment (killbond007) [reply src/shared/feature-flags/flags-status-table.tsx:67]: https://stackoverflow.com/questions/42186723/antd-ui-library-overriding-table-behavior-on-empty-data

here with ConfigProvider we can overriding emptyData in Table .

**Pull request merged**